### PR TITLE
4822 Script tidy up

### PR DESCRIPTION
**Pull request by assoomro**
- names of images and services are now more consistent
- got rid of useless copy statements
- file names are now adhering to our naming convention
- cleaner tear down scripts

### CODE REVIEW

**Comment by assoomro** [comment]
The pipeline needs to be edited slightly to reflect the name changes, otherwise our checks will not pass.

**Comment by talawson05** [comment]
> The pipeline needs to be edited slightly to reflect the name changes, otherwise our checks will not pass.

@assoomro when you say the pipeline needs editing, you mean the azure-pipelines.yml file right?
IF you edit it as part of this PR it will run using your changes, and those changes will only affect you so you won't block anyone else.
Look's like you just want to rename the setup integration DB script task to match your change

**Comment by assoomro** [comment]
> > The pipeline needs to be edited slightly to reflect the name changes, otherwise our checks will not pass.
> 
> @assoomro when you say the pipeline needs editing, you mean the azure-pipelines.yml file right?
> IF you edit it as part of this PR it will run using your changes, and those changes will only affect you so you won't block anyone else.
> Look's like you just want to rename the setup integration DB script task to match your change

Thanks Tony, you're a hero. Didn't know that was an option. I will continue working on this 'nice to have' when/if I get blocked on my current task again :]

File: README.md [hunk 1]
```diff
@@ -226,7 +226,7 @@ To build / update the image run `setup-integration-db` script either in PowerShe
 | CLI | Command |
 |---------------|--------------------|
 |`bash` | `bash setup-integration-db.sh` |
```
Comment (sam-roberts-bjss) [inline README.md]: Has the name of this file changed as well?

**Pull request merged**